### PR TITLE
basic/filesystems: recognize pdfs filesystem as a network filesystem

### PR DESCRIPTION
**Pull request by msekletar**
Fujitsu advises their users to always use _netdev mount option with pdfs mounts. Hence it makes sense to simply consider pdfs mounts as network filesystem mounts.

https://software.fujitsu.com/jp/manual/manualfiles/m130027/j2ul1563/02enz200/j1563-02-06-02-02.html

### CODE REVIEW

**Comment by msekletar** [comment]
@d-hatayama PTAL

File: src/basic/filesystems.c [hunk 1]
```diff
@@ -130,6 +130,7 @@ const FilesystemSet filesystem_sets[_FILESYSTEM_SET_MAX] = {
                 "pvfs2\0"
                 "smb3\0"
                 "smbfs\0"
+                "pdfs\0"
```
Comment (yuwata) [inline src/basic/filesystems.c]: Please sort the entries.

Comment (msekletar) [reply src/basic/filesystems.c]: Good point, sorted!

**Comment by poettering** [comment]
Ist that an open source fs? Shipped in the kernel? Never heard of it?

**Comment by msekletar** [comment]
@poettering it isn't open-source as far as I know.

**Comment by msekletar** [comment]
Hmm, it seems we also need to know the magic value for this FS type (otherwise systemd-analyze filesystems crashes) and I have no idea about that.

**Comment by msekletar** [comment]
@poettering  actually, pdfs is clustered FS and I think we should handle it the same way we deal with glustrefs or lustre. I have updated the PR to reflect that.

**Comment by poettering** [comment]
I am pretty sure we should not add glue code for out-of-kernel closed source stuff explicitly, in particular if it is extremely niche (and I would say it is, given I *never* heard of it before). The _netdev thing exists for a reason, and I think it's totally OK to say that people who use that should set _netdev too. It's not a big ask...

**Comment by msekletar** [comment]
@poettering OK, so let's close this out then.

**Pull request closed without merging**